src/api/state.go

```diff
@@ -40,5 +40,8 @@ func LoadRemoteState(a *StorageApi) (*model.State, *utils.Error) {
 		state.AddError(err)
 	}
 
-	return state, state.Error()
+	if state.Error().Len() > 0 {
```

Comment:
Kontrola kodu tu blbla, a nevedela rozlisit, ze tam moze prist aj `nil` hodnota, tak som to musel prepisat.